### ISSUE TITLE
Painful to navigate to disqus forum on small screen resolution sizes

### ISSUE THREAD

**Issue by AdmiringWorm** [chocolatey/chocolatey.org#703]
### What You Are Seeing?

On small screen laptops there is a problem to scroll down when you are only interested in the disqus section of a certain package.

### What is Expected?

To have a button, or some other way to easily get to the distination (in this case disqus) to see the wanted information of what you need at that moment.

### How Did You Get This To Happen? (Steps to Reproduce)

Well, I don't consider it a bug, maybe an oversight at best. But do the following, set the screen resolution to 1366x768 (don't ask), and try navigating to a package details package with a somewhat long description (bonus points if you only use a touch pad :), package I tried was: https://chocolatey.org/packages/chocolatey). It is doable, but painful without some way to shortcut the the wanted space.

**Comment by ferventcoder** [comment]
Can you define your size of resolution and include an image of what you are seeing? It will make it easier to see what you are seeing.

**Comment by ferventcoder** [comment]
I see screen resolution at 1366x768 in your notes, so ignore that earlier request.

**Comment by ferventcoder** [comment]
@AdmiringWorm is this still a thing given all of the changes that have been made?

**Comment by AdmiringWorm** [comment]
Let me check. Sorry, never got a notification that you added a comment earlier, so completely missed it.

**Comment by AdmiringWorm** [comment]
No, still feels problematic to navigate past the Description of the package.

At all this boils down to the low resolution on the computer though, as only a very small portion of the height of the page is shown at once.

![Screenshot 2019-11-14 at 17 14 57](https://user-images.githubusercontent.com/1474648/68875337-f3cf4e80-0702-11ea-8fc5-b7fadaef1db8.png)
![Screenshot 2019-11-14 at 17 15 39](https://user-images.githubusercontent.com/1474648/68875350-f92c9900-0702-11ea-86d2-715821b0f802.png)
![Screenshot 2019-11-14 at 17 15 46](https://user-images.githubusercontent.com/1474648/68875360-fd58b680-0702-11ea-8e5f-ed341991949a.png)
![Screenshot 2019-11-14 at 17 15 57](https://user-images.githubusercontent.com/1474648/68875370-0184d400-0703-11ea-8c04-f01e5f8d9634.png)

As such, it would be helpful if there was some links/buttons on the top that would allow you to navigate to each section on the page.
